packages/pretty-format/README.md

```diff
@@ -14,11 +14,11 @@ $ yarn add pretty-format
 ## Usage
 
 ```js
-const prettyFormat = require('pretty-format'); // CommonJS
+const {format: prettyFormat} = require('pretty-format'); // CommonJS
 ```
 
 ```js
-import prettyFormat from 'pretty-format'; // ES2015 modules
+import {format as prettyFormat} from 'pretty-format'; // ES2015 modules
```

Comment:
I'm purposefully not using default import here, since real ESM only works with named exports (in node impl) as it's still CJS under the hood

Reply:
I'd go with just `format` in examples, without aliasing to `prettyFormat`. No need for extra mental overhead here

Reply:
it's the difference between "copy the example and run it" vs "copy example, but get weird errors". I think it's worth it to keep as is?